packages/server/api/src/app/app-connection/app-connection-service/oauth2/services/cloud-oauth2-service.ts

```diff
@@ -31,21 +31,32 @@ async function refresh({
     tokenUrl: connectionValue.token_url,
   };
 
-  const oauthProxyUrl = system.get<string>(
-    SharedSystemProp.INTERNAL_OAUTH_PROXY_URL,
-  );
+  logger.info(`Refreshing token via Lambda for block: ${blockName}`);
 
-  const response = (
```

Comment:
so were we just never calling it?

Reply:
we were, but the refresh itself had an empty implementation